inst/book/doublet-detection.Rmd

```diff
@@ -181,13 +181,14 @@ This will use the spike-in size factors to scale the contribution of each cell t
 
 ### Doublet classification  
 
-The `scDblFinder()` function [@germain2020pipecomp] implements a related approach based on doublet simulation.
+The `scDblFinder()` function [@germain2021scDblFinder] combines this artificial doublet density approach with an iterative classification scheme. 
```

Comment:
I don't think I use the words "artificial doublets" anywhere, just "simulated doublets". To be consistent, we should use "simulated" throughout. I would just say something like "combines this simulated doublet density with an iterative...", to avoid having excessively long adjective chains.